packages/peg.rb

```diff
@@ -0,0 +1,19 @@
+# Adapted from Arch Linux peg PKGBUILD at:
+# https://github.com/archlinux/svntogit-community/raw/packages/peg/trunk/PKGBUILD
+
+require 'package'
+
+class Peg < Package
+  description 'recursive-descent parser generators for C'
+  version '0.1.18-2'
+  source_url 'http://piumarta.com/software/peg/peg-0.1.18.tar.gz'
+  source_sha256 '20193bdd673fc7487a38937e297fff08aa73751b633a086ac28c3b34890f9084'
+
+  def self.build
+    system "sed -i 's/PREFIX  = \\/usr\\/local/PREFIX =/g' Makefile"
+    system "PREFIX=#{CREW_PREFIX} make"
+  end
+  def self.install
+    system "make ROOT=#{CREW_DEST_DIR} install"
```

Comment:
You could test with a different `PREFIX` than `/usr/local` but you might need `PREFIX=#{CREW_PREFIX}` here also.

Reply:
Good point there.